package.json

```diff
@@ -15,25 +15,30 @@
     "dist:dir": "yarn dist --dir -c.compression=store -c.mac.identity=null"
   },
   "dependencies": {
-    "source-map-support": "^0.5.16"
+    "source-map-support": "^0.5.21"
   },
+  "//": [
```

Comment:
Remove

Reply:
Are you absolutely 100% sure about that? This is just a comment. It does nothing except tell all contributors: "Don't attempt to update the Electron version unless you also take care to rewrite the entire application to stop using `remote.` calls." It's a good thing to have this comment. I'll remove it if you are sure though.

Reply:
Yes I'm 100% certain about this, developer comments belong in better locations.